repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms.yaml

```diff
@@ -24,3 +24,6 @@ definitions:
     /hippo:configuration/hippo:frontend/cms/cms-validators/blank-attachment-field-forbidden:
       jcr:primaryType: frontend:plugin
       plugin.class: uk.nhs.digital.ps.BlankAttachmentFieldValidator
+    /hippo:configuration/hippo:frontend/cms/cms-validators/blank-granularity-forbidden:
```

Comment:
#26 brings in some namespacing for validators. We should start isolating PS code and config a bit more.

Reply:
I'll wait for #26.

Reply:
Now rebased.